### PR TITLE
Feature preview

### PR DESCRIPTION
**Pull request by piterm11**

### CODE REVIEW

File: src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerPreview.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import edu.kis.legacy.drawer.shape.line.AbstractLine;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
+
+import javax.swing.*;
+import java.util.List;
+
+public class CommandManagerPreview {
+	DrawPanelController controller;
+	LineDriverAdapter lineAdapter;
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerPreview.java]: private!

File: src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java [hunk 2]
```diff
@@ -75,46 +84,50 @@ public CommandManagerWindow(DriverCommandManager commandManager) {
...
 		c.weightx = 10;
 		c.gridx = 1;
-		c.weighty = 20;
+		c.gridy = 0;
+		c.weighty = 1;
+		c.gridheight = 5;
+		c.gridwidth = 1;
 		content.add(previewPanel, c);
-
+		preview = new CommandManagerPreview(previewPanel, new BasicLine());
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java]: dependencies from CommandManagerPreview burden CommandManagerWindow. You didn't manage to inverse dependencies.

**Review by kowallus — CHANGES_REQUESTED** [review]
I cannot see much improvement

**Comment by piterm11** [comment]
Changes have been made

**Comment by pioter00** [comment]
I hope it is finally done.

**Review by kowallus — APPROVED** [review]
Great! 5 pts final

**Pull request merged**